app/views/landing_page/blocks/_blocks_container.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="blocks-container">
```

Comment:
Can you add a some system tests for a block container, similar to the ones for the [hero component](https://github.com/alphagov/frontend/blob/7c2683eecb56c689da1e65c3d09579e54b6e4901/spec/system/landing_page_spec.rb#L37-L43), so that we can test that it's rendering as expected.

You'll also need add a block container block to the fixture file.

Reply:
Fixture updated and system test added as requested.